examples/cholesky.py

```diff
@@ -24,7 +24,7 @@ def cholesky(n, dtype):
     input = np.eye(n, dtype=dtype)
 
     start = time()
-    input.cholesky(no_tril=True)
+    np.linalg.cholesky(input)
```

Comment:
Passing `no_tril=True` here was intentional; unlike LAPACK, NumPy's Cholesky zeroes out the upper triangular matrix, which shouldn't be included when measuring FLOPs, and `no_tril=True` skips that part. Though we want it for measuring performance, we don't want to expose it to the API in order to be consistent with NumPy.

Reply:
@magnatelee apologies, I made a judgment call based on the statement in https://github.com/nv-legate/cunumeric/pull/209#discussion_r818182684 regarding `no_tril` as a parameter that we don't want to expose. I will make a quick follow-up PR today, however, since this is a "public" example, it seems possibly that:

* benchmarking vs public examples could be split out from each other
* `no_tril` should be expose in some manner, after all

Reply:
I agree that `no_tril` needs to be exposed in some way, as we want to keep all our performance tests public. An alternative is to make `np.tril` negligible, so we don't need to worry about it. Let's not make a follow-up PR for now.

Reply:
👍 I will make an issue for now instead